### ISSUE TITLE
Implement device serial query

### ISSUE THREAD

**Issue by Flohack74** [ubports/deviceinfo#2]
For various applications it might be necessary to fingerprint a device. Implement a function that retrieves the device serial number liek the Android getprop equivalent.

**Comment by UniversalSuperBox** [comment]
Some devices might not *have* a serial number burned into read-only space... So it won't be easily possible to get a serial number for them. Do you have a desired usecase in mind?

**Comment by Flohack74** [comment]
No, not sure what was the idea back then. Maybe the long-standing plan to have device hashes to better track usage, reinstallations etc...

**Comment by mariogrip** [comment]
In that case this needs to be a opt in, so the user will choose this on setup example "check this if you want to help collect useful debug information to ubports"... something along those linea. But yeah to have some stats on how stable systems are, crash rapports, and statistics would be really nice for development, but right now i don't think this is a priority.
